test/eval_tests/array.js

```diff
@@ -0,0 +1,10 @@
+'use superstrict';
```

Comment:
This directive is completely ignored for now; add a //TODO to the code so it's explicit that you're not doing anything with it (but we plan to).